core/src/main/java/org/jdbi/v3/core/mapper/EnumMapperFactory.java

```diff
@@ -22,9 +22,11 @@
  */
 class EnumMapperFactory implements ColumnMapperFactory {
     private static final EnumByNameMapperFactory BY_NAME = new EnumByNameMapperFactory();
+    private static final EnumByOrdinalMapperFactory BY_ORDINAL = new EnumByOrdinalMapperFactory();
```

Comment:
Not a blocker for this PR, but what do you think of having `@ByName` and `@ByOrdinal` qualifying annotations which can override the preferred strategy on a case-by-case basis?

Reply:
Yeah, that was brought up before, I'm in favor of it. #1354